docs/README.md

```diff
@@ -0,0 +1,124 @@
+# Bazel Central Registry (BCR) contribution guidelines
+
+Before contributing to the Bazel Central Registry, check the [BCR policies](bcr-policies.md) to learn how the BCR is maintained.
+
+## Contribute a Bazel module
+
+To contribute a new module or a new version to an existing module, you can clone the BCR repository and run the interactive helper script:
+
+```bash
+git clone https://github.com/bazelbuild/bazel-central-registry.git
+cd bazel-central-registry
+python3 ./tools/add_module.py
+```
+
+The script will generate all require changes based on your input, please review, modify and commit the change, then send a PR to the BCR repository.
+
+If you are the project owner, you can set up the [Publish to BCR](https://github.com/apps/publish-to-bcr) Github App for your repository to automatically send a PR to the BCR when cutting a new release.
+
+## Presubmit
+
+Every module version must pass the BCR presubmit before getting merged. The presubmit validate the correctness and consistency of module information, then runs build and test targets specified in the `presubmit.yml` file. The BCR presubmit is driven by the [bcr_presubmit.py](https://github.com/bazelbuild/continuous-integration/blob/master/buildkite/bazel-central-registry/bcr_presubmit.py) script on [Bazel CI](https://github.com/bazelbuild/continuous-integration/tree/master/buildkite#bazel-continuous-integration).
+
+### Validations
+
+Most of presubmit validations are implemented in `./tools/bcr_validation.py`. So you can also run the following command to validate a module locally:
+
+```bash
+python3 tools/bcr_validation.py --check=foo@1.0.0
+```
+
+Validations performed in the scripts are:
+
+- Verify the module version exists in the `metadata.json` of the module.
+- Verify the source archive URL matches the source repository specified in `metadata.json`.
+- Verify the source archive URL is stable if it comes from GitHub. (See [this discussion](https://github.com/bazel-contrib/SIG-rules-authors/issues/11#issuecomment-1029861300))
+- Verify the integrity values of the source archive and patch files (if any) are correct.
+- Verify the checked-in MODULE.bazel file matches the one in the extracted and patched source tree.
+- Check if the module is new or the `presubmit.yml` file is changed compared to the last version, if so a BCR maintainer review will be required to run jobs specified in `presubmit.yml`.
+
+Additional validations implemented in the [bcr_presubmit.py](https://github.com/bazelbuild/continuous-integration/blob/master/buildkite/bazel-central-registry/bcr_presubmit.py) script:
+
+- No checked-in MODULE.bazel, source.json, patches files are not modified in the PR.
```

Comment:
Replace the first "No" with "The" since you have a "not" later; same on the next line

Reply:
Done